bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ChaumPedersen.cs

```diff
@@ -0,0 +1,389 @@
+﻿using System;
+namespace ElectionGuard
+{
+    // Declare native types for convenience
+    using NativeElementModP = NativeInterface.ElementModP.ElementModPHandle;
+    using NativeElementModQ = NativeInterface.ElementModQ.ElementModQHandle;
+    using NativeElGamalCiphertext = NativeInterface.ElGamalCiphertext.ElGamalCiphertextHandle;
+    using NativeDisjunctiveChaumPedersenProof = NativeInterface.DisjunctiveChaumPedersenProof.DisjunctiveChaumPedersenProofHandle;
+    using NativeConstantChaumPedersenProof = NativeInterface.ConstantChaumPedersenProof.ConstantChaumPedersenProofHandle;
+
+    /// <Summary>
+    /// The Disjunctive Chaum Pederson proof is a Non-Interactive Zero-Knowledge Proof
+    /// that represents the proof of ballot correctness (that a value is either zero or one).
+    /// This proof demonstrates that an ElGamal encryption pair (𝛼,𝛽) is an encryption of zero or one
+    /// (given knowledge of encryption nonce R).
+    ///
+    /// This object should not be constructed directly.  Use DisjunctiveChaumPedersenProof::make
+    ///
+    /// see: https://www.electionguard.vote/spec/0.95.0/5_Ballot_encryption/#outline-for-proofs-of-ballot-correctness
+    /// </Summary>
+    public class DisjunctiveChaumPedersenProof : DisposableBase
+    {
+        /// <Summary>
+        /// a0 in the spec
+        /// </Summary>
+        public unsafe ElementModP ZeroPad
+        {
+            get
+            {
+                var status = NativeInterface.DisjunctiveChaumPedersenProof.GetZeroPad(
+                    Handle, out NativeElementModP value);
+                if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
+                {
+                    Console.WriteLine($"ZeroPad Error Status: {status}");
```

Comment:
More a long term comment, these `Console.WriteLine`'s probably should be changed out with some `Exceptions` to help represent the errors better. Perhaps a new issue?